Misc/NEWS.d/next/Core and Builtins/2018-01-17-15-42-56.bpo-32582.kSu6Yj.rst

```diff
@@ -0,0 +1,2 @@
+The :func:`chr` builtin function now catchs :exc:`OverflowError` and raises
```

Comment:
"catches" -- this is an implementation detail. And I think it is better to not raise and catch it.